static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -31,13 +31,16 @@ const setDfpListeners = (): void => {
 
 const setPageTargeting = (): void => {
     const pubads = window.googletag.pubads();
-    const targeting = buildPageTargeting();
+    // because commercialFeatures has the potential to export itself as {}
```

Comment:
Yeah, no idea. I didn't write the module, the exports in it were strange. In fact, it exported nothing when it failed before I made it export an empty object. We could not see what would throw an exception. The old code looked like [this](https://github.com/guardian/frontend/blob/aa-video-embed-tracking/static/src/javascripts/projects/common/modules/commercial/commercial-features.js#L132).